src/test/java/com/serenitydojo/WhenCreatingObjects.java

```diff
@@ -12,4 +12,17 @@ public void creating_a_dog() {
         Assert.assertEquals(fido.getFavoriteToy(), "Bone");
         Assert.assertEquals(fido.getAge(), 5);
     }
+
+    @Test
+    public void dog_makes_noise() {
+        Dog fido = new Dog("Fido", "Bone", 5);
+
+        fido.makeNoise();
+    }
```

Comment:
when makeNoise returns String, you can use assertions here like 

 ```suggestion
        Assert.assertEquals(fido.makeNoise(), "Woof");
    }
```